.github/workflows/publish.yml

```diff
@@ -0,0 +1,23 @@
+name: osc-sdk-rust crate.io publishing
+on:
+  release:
+    types: [released]
+
+jobs:
+  publish:
+    environment: publish
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Stable Rust with rustfmt
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        components: rustfmt
+    - name: Cargo login
+      run: echo $CARGO_TOKEN > cargo login
+      env:
+        CARGO_TOKEN: ${{ secrets.CARGO_TOKEN }}
```

Comment:
I did not tested it, but does it work to use `CARGO_REGISTRY_TOKEN` directly ? (cf https://doc.rust-lang.org/cargo/reference/environment-variables.html)